fe/src/pages/index.vue

```diff
@@ -1,6 +1,41 @@
 <template>
   <div>
-    WELCOME TO VUE!
-    <BaseDropdown />
+    <nav class="h-[64px] border-b grid place-content-center">Navbar</nav>
+
+    <div
+      class="h-[558px] grid grid-cols-3 grid-rows-1 gap-4 text-center bg-gradient-container"
+    >
+      <div class="flex flex-col justify-center items-center">
+        <IconsBlocksLeft />
+      </div>
+      <div class="flex flex-col justify-center items-center gap-[32px]">
+        <h1
+          class="text-[40px] bg-gradient text-transparent bg-clip-text font-bold text-4xl leading-[125%]"
+        >
+          WELCOME TO SUNBLOCK
+        </h1>
+        <p
+          class="font-bold text-base max-w-[479px] text-center leading-[125%] text-[18px]"
+        >
+          Contribute to projects and innovations that matter using Blockchain
+          technology.
+        </p>
+        <div>
+          <p class="text-base mb-[8px]">Want to make a project?</p>
```

Comment:
Make use of tailwind's default class values.
```suggestion
          <p class="text-base mb-2">Want to make a project?</p>
```